packages/bitcore-client/bin/wallet-paypro

```diff
@@ -80,56 +81,68 @@ const main = async () => {
   // create transaction
   const { instructions } = paymentRequestResponse;
   console.log(JSON.stringify(instructions, null, 2));
-  const instruction = instructions[0];
-  const { type, requiredFeeRate, outputs, gasPrice, to, value, data } = instruction;
-  const recipients = [{
-    address: to || outputs[0].address,
-    amount: value || outputs[0].amount
-  }];
-  const fee = requiredFeeRate || gasPrice;
-  let invoiceID;
-  if (outputs) {
-    invoiceID = outputs[0].invoiceID;
-  }
-  const utxos = await wallet.getUtxosArray();
-  const params = { recipients, fee, from, invoiceID, data, value, utxos };
-  params.nonce = Number(nonce);
-  const unsignedTx = await wallet.newTx(params);
-  console.log(`unsignedRawTx: ${unsignedTx}`);
-
-  //  Sign tx
-  const password = await promptly.password('Wallet Password:');
-  wallet = await wallet.unlock(password);
-  const signedTx = await wallet.signTx({ tx: unsignedTx, passphrase: password });
-  const txid = cwc.Transactions.getHash({chain, tx: signedTx});
-  console.log(`signedRawTx: ${signedTx}`);
-  console.log(`txid: ${txid}`);
+  let unsignedTransactions = [];
+  let signedTransactions = [];
+  nonce--;
```

Comment:
seems like the nonce incrementing should be at the end of the loop, so you don't have to -- and then ++